### DISCUSSION TITLE
Web scrapper error - TypeError: (intermediate value) is not iterable (cannot read property undefined

### DISCUSSION THREAD

**Discussion by Shankythe1** [FlowiseAI/Flowise#2072] [Q&A]
Can anyone help me how to resolve this error, im new to Flowise, not sure how to get thru this error. Any help will be greatly appreciated. thanks!

**Comment by mkmatzat** [comment]
I have the same issue with the OpenAI Chat Model. If I uses Google Vertexx AI instead it works with the same settings for the other artefacts.

**Comment by MarouaneBenbetka** [comment]
can you provide the code, how did you get that error , 

cause this error usually occurs when you try to destructure a value that is not iterable, like null or undefined

**Reply by mkmatzat** [reply]
I have no code written, I only dragged and dropped a flow toghether and then used the Flowise chat widget to ask a question:

![image](https://github.com/FlowiseAI/Flowise/assets/166175104/3016d443-0ae5-43f0-9c62-175b013e1ef6)

**Reply by HenryHengZJ** [reply]
Can you attach your exported flow?

**Comment by mkmatzat** [comment]
Attached is my the affected flow.
[Mathias Broker Experte Chatflow.json](https://github.com/FlowiseAI/Flowise/files/14967162/Mathias.Broker.Experte.Chatflow.json)

This is tested and fails on 1.6.3 running on a Railway server. On my laptop with Windows 11 it works fine.
